ledger-service/cli-opts/src/main/scala/cliopts/Logging.scala

```diff
@@ -34,4 +57,24 @@ object Logging {
       )
     ()
   }
+
+  def useJsonLogEncoderParse[C](
+      parser: scopt.OptionParser[C]
+  )(useJsonLogEncoder: Setter[C, Boolean]): Unit = {
```

Comment:
Maybe introduce a separate type for the supported logging formats instead of using a boolean? That makes it easier to not swap the meaning of the boolean accidentally.

Reply:
done